chapters/introduction.tex

```diff
@@ -30,94 +39,116 @@ \section{Doing credible research at scale}
+US\$180 million research budget to shape the design and implementation of 
+US\$18 billion in development finance. 
+DIME also provides advisory services to 30 multilateral and bilateral development agencies. 
+Finally, DIME invests in public goods to improve the quality and reproducibility of development research around the world. 
+
+DIME Analytics was created to take advantage of the concentration and scale of research at DIME to develop and test solutions, 
+to ensure high quality of data collection and research across the DIME portfolio, 
+and to make public training and tools available to the larger community of development researchers.
+Data for Development Impact compiles the ideas, best practices and software tools Analytics 
+has developed while supporting DIME's global impact evaluation portfolio. 
+The \textbf{DIME Wiki} is one of our flagship products, a free online collection of our resources and best practices.\sidenote{
+\url{http://dimewiki.worldbank.org/}}
+
+This book complements the DIME Wiki by providing a structured narrative of the data workflow for a typical research project. 
 We will not give a lot of highly specific details in this text,
-but we will point you to where they can be found,
-and give you a sense of what you need to find next.
-Each chapter will focus on one task,
-and give a primarily narrative account of:
+but we will point you to where they can be found.\sidenote{Like this: 
+\url{https://dimewiki.worldbank.org/wiki/Primary_Data_Collection}}
+Each chapter focuses on one task, providing a primarily narrative account of:
 what you will be doing; where in the workflow this task falls;
-when it should be done; who you will be working with;
-why this task is important; and how to implement the most basic form of the task.
-
-We will use broad terminology throughout this book
-to refer to different team members:
+when it should be done; and how to implement it according to best practices.
+
+
+
+\section{Outline of this book}
+The book progresses through the typical workflow of an empirical research project.
+We start with ethical principles to guide empirical research, 
+focusing on research transparency and the right to privacy. 
+The second chapter discusses the importance of planning data work at the outset of the research project- 
+long before any data is acquired - and provide suggestions for collaborative workflows and tools. 
+Next, we turn to common research designs for \textbf{causal inference}{\sidenote{causal inference: identifying the change in outcome \textit{caused} by a particular intervention}}, and consider their implications for data structure. 
+The fourth chapter covers how to implement sampling and randomization to ensure research credibility, 
+and includes details on power calculation and randomization inference. 
+The fifth chapter provides guidance on high quality primary data collection, particularly for projects that use surveys. 
```

Comment:
Is this still the best way to describe ch5 after @bbdaniels restructuring in https://github.com/worldbank/d4di/issues/351 ?

Reply:
Probably not! noting in #351 that we should update once rewrite is complete.